tests/regression_tests/dagmc/external/test.py

```diff
@@ -101,15 +101,25 @@ def __init__(self, executable, statepoint_name, model):
         self.executable = executable
 
     def _run_openmc(self):
+    """
```

Comment:
Thank you for adding this @helen-brooks! Very helpful. Just gotta indent it so the interpreter recognizes it as a docstring of the function.